variables.tf

```diff
@@ -55,6 +55,16 @@ variable "enable_dns_support" {
   default     = false
 }
 
+variable "enable_classiclink" {
+  description = "should be true if you want to use ClassicLink within the VPC"
+  default     = true
```

Comment:
Keep this default to `false` to be backward compatible, also Terraform uses `false` as default.

Reply:
Changed.